src/app/contextBroker/contextBroker.cpp

```diff
@@ -1554,6 +1561,14 @@ int main(int argC, char* argV[])
 
   while (1)
   {
-    sleep(10);
+    if (subCacheInterval != 0)
+    {
+      sleep(subCacheInterval);
+      // orion::subCache->refresh();
```

Comment:
I though that activating the line make some func test to fail... but probably I have understand you wrongly. Thus, NTC